.ci/scripts/install-kubernetes-autodiscover-test-dependencies.sh

```diff
@@ -17,7 +17,7 @@ MSG="parameter missing."
 HOME=${HOME:?$MSG}
 
 KIND_VERSION="v${KIND_VERSION:-"0.20.0"}"
```

Comment:
Kind is v1.27.3 I think this upgrade

Reply:
[Here](https://github.com/kubernetes-sigs/kind/releases) it says the latest version for kind is 0.20.0, maybe you are mixing up with the kindest version for the cluster? @gizas

Reply:
Ah yes maybe seen the node version in my local tests. So Ignore this comment